apps/wallet-mobile/jestSetup.ts

```diff
@@ -31,6 +31,34 @@ jest.mock('react-native-reanimated', () => {
   Reanimated.default.call = () => {}
   return Reanimated
 })
+jest.mock('react-native-localize', () => ({
```

Comment:
Docs say to use
```tsx
import localizeMock from "react-native-localize/mock";

jest.mock("react-native-localize", () => localizeMock);
```

But our jest complains importing esm, says `unexpected export` stuff.
Tried with require, wouldn't work either.
So I just copied all the mocked functions here.